www/js/lib/uiUtil.js

```diff
@@ -227,12 +228,83 @@ define([], function() {
             '/': '&#x2F;',
             '`': '&#x60;',
             '=': '&#x3D;'
-          };
-          string = String(string).replace(/[&<>"'`=/]/g, function (s) {
-            return escapechars[s];});
-          return string;
+        };
+        string = String(string).replace(/[&<>"'`=/]/g, function (s) {
+            return escapechars[s];
+        });
+        return string;
+    }
 
+    /**
+     * Removes the animation effect between various sections
+     */
+    function removeAnimationClasses() {
+        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
+        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
+        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
     }
+    
+    /**
+     * Adds the slide animation between deifferent sections
+     * 
+     * @param {String} name_of_section It takes the name of the section to which the animation is to be added
+     * 
+     */
+    function addAnimation(section) {
+        if (section == 'home') {
```

Comment:
The content of this function might be simplified and be more generic to make it easier to add another section. But it's not crucial, you can leave it this way if you don't see how to do it

Reply:
@mossroy 
I have made this more generic by adding switch statement .Now new sections can easily be added....Please have a look

Reply:
I am sorry that  I am not very clear about this...So please clarify.